src/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -97,19 +98,18 @@ public async Task<bool> ProcessPayload(WorkflowRequestEvent message, Payload pay
                 await _workflowRepository.GetWorkflowsByAeTitleAsync(message.CalledAeTitle) as List<WorkflowRevision> :
                 await _workflowRepository.GetByWorkflowsIdsAsync(message.Workflows) as List<WorkflowRevision>;
 
-            if (workflows is null)
+            if (workflows is null || workflows.Any() is false)
             {
                 return false;
             }
 
             var workflowInstances = new List<WorkflowInstance>();
 
-            foreach (var workflow in workflows)
-            {
-                var workflowInstance = await CreateWorkflowInstanceAsync(message, workflow);
-                workflowInstances.Add(workflowInstance);
-            }
+            var tasks = workflows.Select(workflow => CreateWorkflowInstanceAsync(message, workflow));
```

Comment:
would this be simpler as 
            var tasks = await workflows.Select(async workflow => await CreateWorkflowInstanceAsync(message, workflow));
            
            I think is the syntax?